data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+			variant
+				"Albatross (Shield)"
+				"Albatross"
+				"Ibis" 2
+				"Starling (Heavy)" 6
+				"Petrel" 4
+				"Tern" 8
+	on enter "Aescolanus"
+		dialog `The Korath fleet is gone. Given the urgency Prefect Chilia put on this mission, the onus is on you to find it... Quickly.`
+	on complete
+		conversation
+			`The taskforce streams back into Caelian without fanfare. The ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they just did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`
+
+
+
+mission "Remnant: Cognizance 5"
+	name "Transport Plume to a meeting"
+	description "Prefect Chilia needs Plume to consult about the void sprites."
+	deadline 5
+	source "Caelian"
+	stopover "Aventine"
+	to offer
+		has "Remnant: Cognizance 4: done"
+	on offer
+		conversation
+			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`
+			`	"So, what should we do differently next time?"`
+			choice
+				`	"A few more Albatrosses would have been great."`
+				`	"The EMPs kept getting in the way."`
+				`	"I think that went fairly well, actually."`
+			`	He writes something on his notepad. "I appreciate the feedback." He pauses, and sips from his glass. "Now, moving forward, we need to know what the Korath are doing. Beyond the obvious of killing void sprites and picking fights with the Archon. The Korath are a cunning foe, and they would not antagonize the Archon for no reason." He looks thoughful.`
+			`	"I have to look into a few things here. In the meantime, please go pick up Plume from Aventine, and bring him back. We need him to be here for a briefing before <date>."`
+				accept
+	on stopover
+		dialog
+			`Plume is wating for you at the landing pad. Once you have given the all-clear and opened the doors, he boards with little fanfare. "Thanks for dealing with the Korath, captain <last>. They have me worried." With that, he settles into a bunk and starts reviewing his notes.`
```

Comment:
`Plume is waiting for you at the landing pad. Once you have given the all-clear and opened the doors, he boards with little fanfare. "Thanks for dealing with the Korath, captain <last>. They had me worried." With that, he settles into a bunk and starts reviewing his notes.`

Reply:
fixed.